packages/pwa-kit-react-sdk/src/ssr/universal/utils.js

```diff
@@ -51,3 +53,59 @@ export const getProxyConfigs = () => {
     // Clone to avoid accidental mutation of important configuration variables.
     return configs.map((config) => ({...config}))
 }
+
+export const DEFAULT_CONFIG_MODULE_NAME = 'DEFAULT'
+
+/**
+ *
+ * @param {*} param0
+ * @returns
+ */
+export const getConfig = (opts = {}) => {
+    const {fileNameResolver} = opts
+    if (_config) {
+        return _config
+    }
+
+    if (typeof window !== 'undefined') {
+        // NOTE: At this point in time the serialized data has not been assigned to the global,
+        // so we have to get it directly from the markup. :(
+        _config = JSON.parse(document.getElementById('mobify-data').innerHTML).__APPCONFIG__
+
+        return _config
+    }
+
+    const _require = eval('require')
+    const {cosmiconfigSync} = _require('cosmiconfig')
+
+    // Load the config synchronously using a custom "searchPlaces".
+
+    // By default use the deplayment target as the {moduleName} for your
+    // configuration file. This means that on a "Production" names target, you'll load
+    // your `config/production.json` file. You can customize how you determine your
+    // {moduleName}.
+    let moduleName = fileNameResolver && fileNameResolver()
+    moduleName = process?.env?.DEPLOY_TARGET || DEFAULT_CONFIG_MODULE_NAME
+
+    // TODO: Move `searchPlaces` out of this function. Don't forget to fix the other
+    // unknown place right now where we look for proxy information.
+    const explorerSync = cosmiconfigSync(moduleName, {
+        packageProp: 'mobify',
+        searchPlaces: [
+            `config/${moduleName}.yml`,
+            `config/${moduleName}.yaml`,
+            `config/${moduleName}.json`,
+            `config/local.yml`,
+            `config/local.yaml`,
```

Comment:
The moduleName will always be `default` when there is no resolver or `DEPLOY_TARGET` is undefined. In this case, the local.json will always be ignored. We probably don't really need `DEFAULT_CONFIG_MODULE_NAME` imo

Reply:
Good catch.. resolved it.